server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -160,20 +81,17 @@ public Builder dynamic(Dynamic dynamic) {
             return this;
         }
 
-        public Builder nested(Nested nested) {
-            this.nested = nested;
-            return this;
-        }
-
         public Builder add(Mapper.Builder builder) {
             mappersBuilders.add(builder);
             return this;
         }
 
-        @Override
-        public ObjectMapper build(ContentPath contentPath) {
-            contentPath.add(name);
+        public List<Mapper.Builder> builders() {
+            return mappersBuilders;
+        }
 
+        protected Map<String, Mapper> buildMappers(ContentPath contentPath) {
```

Comment:
Small comment, would be nice to make this `protected final` to clarify it's not meant to be overridden.